src/App.java

```diff
@@ -1,23 +1,68 @@
 package application;
 
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
 public class App {
 
+
   public static void main(String[] args) {
+    Employee[] array = new Employee[2];
+    int count = 0;
+    do{
+      if (count==array.length) array=newArray(array);
+      array[count]=createEntries();
+      count++;
+    }while(exit());
+   printList(array);
+  }
+
 
+  private static Employee[] newArray(Employee[] oldArray) {
+    Employee[] newArray = new Employee[oldArray.length*2];
+    newArray= Arrays.copyOf(oldArray,oldArray.length);
+   return newArray;
   }
 
-  /*
-  * for the bonus task
-  *
-  * private static int generateID() {}
-  *
-  */
+  private static void printList(Employee[] array){
+    System.out.println("Folgende Mitarbeiter wurden angelegt");
+    for (int i = 0; i  < array.length ; i ++) {
+      System.out.println(array[i].getId()+" "+array[i].getFirstname()+" "+array[i].getLastname());
+    }
+    System.out.println("Auf Wiedersehen!");
+  }
 
-  /*
-   * for the console application
-   *
-   * private static void application() {}
-   *
-   */
+  private static int generateID(){
+    Random random=new Random();
+    return random.nextInt(1000);
+  }
+  private static boolean exit(){
+    char stopp;
+    Scanner scanner=new Scanner(System.in);
+    System.out.println("x für beenden. Beliebige Taste für weiter");
+    stopp=scanner.next().charAt(0);
```

Comment:
Refering the comment above

~~~java
char stopp = scanner.next().charAt(0);
~~~